make_UPD_igv.py

```diff
@@ -0,0 +1,176 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def parse_ped(ped_file):
+    samples = {}  # 'sample_id': {'family': 'fam_id', 'parents': ['sample_id', 'sample_id']}
+    for line in ped_file:
+        ped_data = line.strip().split()
+        family, sample, father, mother, sex, phenotype = ped_data
+
+        # Create samples
+        if sample not in samples:
```

Comment:
samples is defined at line 7 as `samples = {}`
so... sample is never in samples?

Reply:
EDIT: aah, for the first line, you take the headers?

Reply:
no, I still do not get it. afterwards you still ask `not in samples`

Reply:
Statement is needed if child is present in PED file before parents. 
Otherwise, child would not be assigned in the parent(s).